src/appleseed.studio/mainwindow/mainwindow.cpp

```diff
@@ -514,6 +526,8 @@ void MainWindow::build_log_panel()
 
 void MainWindow::build_python_console_panel()
 {
+    PythonInterpreter::instance().set_mainwindow(this);
```

Comment:
Can we move this to the place where the mainwindow is created?
It's not really related to the python console panel.